### DISCUSSION TITLE
How to restore a backup? It's not mentioned in the docs.

### DISCUSSION THREAD

**Discussion by woutersamaey** [btcpayserver/btcpayserver#2928] [General]
The docs tell us how to backup on page https://docs.btcpayserver.org/Docker/#how-can-i-back-up-my-btcpay-server, but it does not mention how to do a restore.

Backup up without a restore procedure isn't very handy.

Can anyone comment on this so we can improve the docs?

**Comment by woutersamaey** [comment]
@pavlenex can we get this on the roadmap?

**Comment by woutersamaey** [comment]
One idea is that we could have a "Restore a previous BTCPay Server backup" option during the configurator wizard.

**Comment by pavlenex** [comment]
Opened a new issue for this one https://github.com/btcpayserver/btcpayserver/discussions/2928 will lock and archive this discussion since naything related to backup documentation process will be handled in that issue.
